lib/replace/js.js

```diff
@@ -41,7 +41,24 @@ const replaceJs = (code, espreeOptions) => {
 
   traverse(ast, {
     pre: (node) => {
-      if (node.type === 'Literal' && typeof node.value === 'string') {
+      // Avoid recursing into a "in" node since it can't be a CSS class or variable.
```

Comment:
This seems to come from another PR

Reply:
Oups. 😢 
I was working on this at the same time as the cuckoo stuff, and I mixed my branches.